PWGGA/GammaConvBase/AliConvEventCuts.cxx

```diff
@@ -117,6 +117,7 @@ AliConvEventCuts::AliConvEventCuts(const char *name,const char *title) :
   fGeneratorNames(NULL),
   fPeriodEnum(kNoPeriod),
   fEnergyEnum(kUnset),
+  fTimeRangeCut(),
```

Comment:
can you please check if this throws a warning during compilation?

Reply:
checked, does not throw a warning